src/requestor/routes/maintain.py

```diff
@@ -97,6 +100,29 @@ async def create_request(
         logger.debug(f"Got username from token: {token_username}")
         data["username"] = token_username
 
+    if "revoke" in api_request.query_params:
+        if api_request.query_params["revoke"]:
```

Comment:
Would this condition work if the url says `<url>?revoke=False`? Also should it work?

Reply:
Nevermind. I think [this](https://github.com/uc-cdis/requestor/blob/cf7132c612ff013bc8b1185dae8c168f887d56fa/tests/test_maintain.py#L413-L413) unit test answers that question.

Reply:
right, i made it return 400 if there is a value so i don't have to interpret all the variations of True, true, t, False, false, f, None, "", etc